Flashcards.Dejmenek/Flashcards.Dejmenek/Controllers/StacksController.cs

```diff
@@ -0,0 +1,115 @@
+﻿using Flashcards.Dejmenek.DataAccess.Repositories;
+using Flashcards.Dejmenek.Helpers;
+using Flashcards.Dejmenek.Models;
+using Flashcards.Dejmenek.Services;
+
+namespace Flashcards.Dejmenek.Controllers
+{
+    public class StacksController
+    {
+        private readonly StacksRepository _stacksRepository;
+        private readonly UserInteractionService _userInteractionService;
+        public Stack? CurrentStack { get; private set; }
+
+        public StacksController(StacksRepository stacksRepository, UserInteractionService userInteractionService)
+        {
+            _stacksRepository = stacksRepository;
+            _userInteractionService = userInteractionService;
+        }
+
+        public void AddStack()
+        {
+            string name = _userInteractionService.GetStackName();
+
+            while (_stacksRepository.StackExistsWithName(name))
+            {
```

Comment:
I expected to get a message saying that germany already existed or the stack I was trying to add already existed
![image](https://github.com/TheCSharpAcademy/CodeReviews.Console.Flashcards/assets/57032138/9c15c2f2-4dba-4e0e-a206-7d31976f8e03)

Can you add a message so that it will be user friendly? 😊